tests/redirects_tests/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from .views import normal_view
+
+urlpatterns = [
+    url(r'^$', normal_view),
```

Comment:
I think you could skip the views file and use `lambda req: HttpResponse('OK')` instead.